### PR TITLE
chore(recipients-app): Bump android versions and dependencies

### PR DESCRIPTION
**Pull request by MDikkii**
So... I tried to run the app with Android Studio Ladybug installed and made the changes until the build worked again.

The one problem with this solution is that `Sentry` seems to be crashing at the start. That means that we may lose the sentry logs if it can't initialize properly. 

## Release Notes

- **New Features**
	- Enhanced localization support with new tooltip and date label methods.
  
- **Improvements**
	- Updated application namespace and ID for better identification.
	- Upgraded to Gradle 8.5 and updated plugin versions for improved compatibility.
	- Updated `intl` dependency to version 0.19.0 for potential new features and fixes.

- **Bug Fixes**
	- Adjusted button styling logic for consistent appearance across states. 

- **Chores**
	- Removed unnecessary text scaling from the MediaQuery widget in tests.

### CODE REVIEW

File: recipients_app/test/helpers/golden_test_device_scenario.dart [hunk 1]
```diff
@@ -27,7 +27,6 @@ class GoldenTestDeviceScenario extends StatelessWidget {
             size: device.size,
             padding: device.safeArea,
             platformBrightness: device.brightness, 
-            textScaler: TextScaler.linear(device.textScaleFactor),
```
Comment (KarinBerg) [inline recipients_app/test/helpers/golden_test_device_scenario.dart:30]: Why did you removed this line? Did you run the tests and then it failed and you had to remove this line?

File: recipients_app/pubspec.yaml [hunk 2]
```diff
@@ -25,7 +25,7 @@ dependencies:
     sdk: flutter
   flutter_native_splash: ^2.4.0
 
-  intl: ^0.18.0
```
Comment (KarinBerg) [inline recipients_app/pubspec.yaml:28]: I think you are using the wrong Flutter version. We are still on 3.19.6

**Review by KarinBerg — CHANGES_REQUESTED** [review]
See Slack for why you had to do all this changes :)

File: recipients_app/test/helpers/golden_test_device_scenario.dart (hunk 1 above)
Comment (MDikkii) [reply recipients_app/test/helpers/golden_test_device_scenario.dart:30]: No, new version of the dependency just show that there is no more TextScaler 🤔

**Pull request closed without merging**